tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -368,12 +390,35 @@ def mirror_status(self, *args):
 
     # Add Peer
     def peer_add(self, **kw):
-        return self.exec_cmd(
-            cmd="rbd mirror pool peer add {} {}".format(
-                kw.get("poolname"), kw.get("cluster_spec")
+            return self.exec_cmd(
+                    cmd="rbd mirror pool peer add {} {}".format(
+                        kw.get("poolname"), kw.get("cluster_spec")
+                    )
             )
+    
+    #Create bootstrap token on primary cluster
+    def bootstrap_peers(self, **kw):
+        return self.exec_cmd(
+                cmd="rbd mirror pool peer bootstrap create --site-name master {} > /root/bootstrap_token_master".format(
+                    kw.get("poolname")
+                )
         )
 
+    #Import bootstrap token from scecondary cluster
+    def import_bootstrap(self, **kw):
+        if "one-way" in kw.get("way", ""):
+            return self.exec_cmd(
+                    cmd="rbd mirror pool peer bootstrap import --site-name slave --direction rx-only {} /root/bootstrap_token_master".format(
+                        kw.get("poolname")
```

Comment:
as `poolname` is a mandatory attribute, we can fail fast by using `kw["poolname"]`. In this case, we raise an attribute error instead of performing the costly operation of going and executing a failure... (The case when poolname is None or not passed)